### PR TITLE
cgroup:Fix CPU utilization of cfs_bandwidth's issues

### PR DESCRIPTION
**Pull request by xiangchunfu**
1.Fix VMStress function missing parameter's issue.
2.Fix qemu process control CPU utilization's issue.

id: 1770058
Signed-off-by: Xiangchun Fu <xfu@redhat.com>

### CODE REVIEW

**Comment by xiangchunfu** [comment]
(1/1) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.cgroup.cfs_bandwidth: PASS (190.28 s)

**Comment by xiangchunfu** [comment]
@xutian Could you help review this thanks?

File: qemu/tests/cfg/cgroup.cfg [hunk 1]
```diff
@@ -96,5 +96,5 @@
         - cfs_bandwidth:
             start_vm = "no"
             check_install_stress = "which stress"
-            stress_cmd = "stress -c 1 -t 100"
+            stress_cmd = "stress -c 4 -t 200"
```
Comment (xutian) [inline qemu/tests/cfg/cgroup.cfg]: values of -c "option" should match with  vcpu counts but not static value in here, right?

File: qemu/tests/cgroup.py [hunk 2]
```diff
@@ -2112,7 +2112,7 @@ def stress_thread():
             time.sleep(10)
             o = process.system_output("top -b -p %s -n 3 | tail -1" % vm.get_pid(),
                                       shell=True).decode()
-            if 49 <= float(o.split()[-4]) <= 51:
+            if 43 <= float(o.split()[-4]) <= 51:
```
Comment (xutian) [inline qemu/tests/cgroup.py]: limited cpu time 50%, so we just need to check cpu utilization less than 51% is enough

File: qemu/tests/cfg/cgroup.cfg (hunk 1 above)
Comment (xutian) [reply qemu/tests/cfg/cgroup.cfg]: 400%  overcommit maybe caused your  system hang

File: qemu/tests/cgroup.py (hunk 2 above)
Comment (xiangchunfu) [reply qemu/tests/cgroup.py]: Updated

File: qemu/tests/cfg/cgroup.cfg (hunk 1 above)
Comment (xiangchunfu) [reply qemu/tests/cfg/cgroup.cfg]: updated,  As vcpu number is 1 for this case. so use '-c 1'  (static) option.

**Comment by xutian** [comment]
Acked-by: Xu Tian <xutian@redhat.com>

**Comment by xutian** [comment]
please ask feature owner to double check.

**Comment by xiangchunfu** [comment]
Double checked and re-run this case.  I got expect result.  It makes sense.

 (1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.cgroup.cfs_bandwidth: PASS 

CPU utilization of guest is: 49.20%
CPU utilization of guest is: 49.30%
CPU utilization of guest is: 49.80%
CPU utilization of guest is: 49.20%
CPU utilization of guest is: 49.80%
CPU utilization of guest is: 49.50%

**Pull request merged**